libcrux-ml-dsa/src/simd/traits.rs

```diff
@@ -30,6 +30,16 @@ pub(crate) trait Repr: Copy + Clone {
 pub(crate) mod specs {
```

Comment:
Thanks for moving all the proof code in here. Since this is huge by now, let's move it into a separate file to keep things a little cleaner.

Reply:
The big zeta functions will likely move to the Spec module, so let's leave that for a future PR.
I would prefer the pre and post functions to remain close to the trait, and they are not very big.